### ISSUE TITLE
Add or clarify support for TimescaleDB 2.1.0 on Postgres 13

### ISSUE THREAD

**Issue by randombk** [timescale/promscale#542]
It looks like Promscale [does not support Postgres 13](https://github.com/timescale/promscale/blob/e7f50e2345301e1be77fa3e34c9359742e254f27/pkg/version/version.go#L45), failing with the error when connecting to a database using the official docker image `timescale/timescaledb:2.1.0-pg13`:

```
level=error ts=2021-03-20T01:34:28.763Z caller=runner.go:43 msg="aborting startup due to error" err="dependency error: problem checking PostgreSQL version: Incompatible postgresql version. Supported server version =12.x"
```

This lack of support should either be addressed or mentioned in the README. Thanks!

**Comment by Pizzahd88** [comment]
Unfortunately I too realized that only pg12 were supported too late. It would have been nice to know it up front before upgrading the entire setup.

**Comment by Harkishen-Singh** [comment]
I think our next release (should be in a day or two) with version `0.3.0` fixes this.

Please correct me if I am wrong @cevian.

**Comment by andrejpodzimek** [comment]
This is quite confusing indeed. For me the `promscale_extension` thing builds and loads and appears to be compatible with PostgreSQL 13, but `promscale` itself fails exactly as reported.

**Comment by Harkishen-Singh** [comment]
I guess we have updated the promscale_extension with PG 13 compatibility on https://github.com/timescale/promscale_extension/pull/13. Promscale is yet to be updated for PG13 and the usage with extension will happen in the upcoming release (0.3.0).

**Comment by cevian** [comment]
Yes this should be fixed in the release tomorrow. We plan to add dependency information into our release notes going forward.

**Comment by Harkishen-Singh** [comment]
@randombk now that we have released the 0.3.0 version which supports PG13, can you please retry with the newer release and post the update here?

**Comment by cevian** [comment]
Closing this issue as I believe it is resolved in 0.3.0 and we have added documentation in release notes. If there are still issues, feel free to reopen the issue.
